### DISCUSSION TITLE
Guide for hosting foundryvtt-docker on a Synology NAS

### DISCUSSION THREAD

**Discussion by vlaquest** [felddy/foundryvtt-docker#265] [Show and tell]
Hi,
I made a complete guide for hosting this image a Synology NAS.
There is no command line involved:
https://foundryvtt.wiki/en/setup/hosting/Synology#setting-up-with-docker-no-command-line

Thanks for this very useful work!

**Comment by KeithPepin** [comment]
@vlaquest : Thank you so much for the detailed Guide. Unfortunately for me, Synology has changed their container management software (basically a wrapper for Docker). As someone with far less knowledge, I'm having trouble translating your guide's instructions at certain points to the Container Manager software they now use due to the changes in labels and UI elements.  Any chance you could update the guide to account for this?

**Reply by vlaquest** [reply]
Hi, I'm sorry it is really not planned at the moment...
It was cool to run foundry VTT in a docker container, but I recognize that it is also quite easy to run it directly on the NAS, all it requires is nodeJS, and this will also allow self updates... As there is no serious support for docker images, they won't be updated as often as the software itself. I can't really recommend to run foundry VTT in docker. 
All you need is a trick to make it restart when it eventually crashes or to run it at NAS startup. Something like pm2 or systemd, but I'm definitely no linux man, and I'm not fully happy with either of those solutions. I run it "manually"  with a background command.
Good luck !

**Comment by KeithPepin** [comment]
Thanks, I appreciate the hard earned knowledge. I’ve got node.js already on
the NAS so I’ll look into options for running it straight up.  I’m brand
new to foundry VTT, so I’m hoping all the effort is worth it.

Keith

*Keith Pepin*
